dept44-starter-webservicetemplate/src/main/java/se/sundsvall/dept44/configuration/webservicetemplate/WebServiceTemplateBuilder.java

```diff
@@ -244,78 +247,74 @@ private void setMessageFactory(final WebServiceTemplate webServiceTemplate) {
 	}
 
 	private void setHttpComponentsMessageSender(final WebServiceTemplate webServiceTemplate) {
-		HttpComponentsMessageSender httpComponentsMessageSender = new HttpComponentsMessageSender();
-
-		// Only use SSL if we have a keystore and password
-		if (shouldUseSSL()) {
-			httpComponentsMessageSender.setHttpClient(createHttpClientWithSSL());
-		} else { // Otherwise provide default
-			httpComponentsMessageSender.setHttpClient(createHttpClientWithoutSSL());
-		}
+		var httpComponents5MessageSender = new HttpComponents5MessageSender();
 
 		if (shouldUseBasicAuth()) {
-			httpComponentsMessageSender.setCredentials(
-				new UsernamePasswordCredentials(basicAuthentication.username(), basicAuthentication.password()));
+			httpComponents5MessageSender.setCredentials(
+				new UsernamePasswordCredentials(basicAuthentication.username(), basicAuthentication.password().toCharArray()));
 		}
 
-		webServiceTemplate.setMessageSender(httpComponentsMessageSender);
-	}
-
-	private HttpClient createHttpClientWithSSL() {
-		SSLContext sslcontext;
-		try {
-			sslcontext = SSLContexts.custom()
-				.loadTrustMaterial(getKeyStore(), (chain, authType) -> true)
-				.loadKeyMaterial(getKeyStore(), keyStorePassword.toCharArray())
-				.build();
-		} catch (Exception e) {
-			throw new RuntimeException("Couldn't load keystore", e);
-		}
-
-		SSLConnectionSocketFactory sslConnectionFactory = new SSLConnectionSocketFactory(sslcontext, NoopHostnameVerifier.INSTANCE);
-
-		final HttpClientBuilder httpClientBuilder = getDefaultHttpClientBuilder().setSSLSocketFactory(sslConnectionFactory);
+		// Only use SSL if we have a keystore and password
```

Comment:
Is this comment still accurate after removal of shouldUseSSL?

Reply:
No, it was a bit out-of-place but it's changed now